seq2science/scripts/deseq2/run_as_rule.R

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env Rscript
-
```

Comment:
why did you remove this?

Reply:
R's `source()` functions reads the sourced file inline. All objects and libraries are globals. It also means we only need to specify the shebang on the script(s) that are supposed to be called from the command line (pretty sure, I'll test this before merging). These files are supposed to be called from deseq.R only, never alone.